### PR TITLE
Noticias image

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: src/native/components/noticias/NoticiaItem.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    borderBottomWidth: 1,
+    borderStyle: 'solid',
+    borderBottomColor: 'black',
+  }
+});
+
+const NoticiaItem = ({item}) => (
+  <View style={styles.bordered}>
+    <Spacer size={15} />
+    <Text style={{ fontWeight: '800' }}>{item.title} • {distanceInWordsToNow(item.publishDate)}</Text>
```
Comment (varffs) [inline src/native/components/noticias/NoticiaItem.js:23]: I guess somehow we should make stuff like this into generic display components?

File: src/native/components/noticias/NoticiaItem.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    <Spacer size={15} />
+    <View>{getRNDraftJSBlocks({ contentState: JSON.parse(item.rawContent) })}</View>
+    <Spacer size={15} />
+    { item.images !== undefined ?
```
Comment (varffs) [inline src/native/components/noticias/NoticiaItem.js]: Lol is this the same error as on the admin? Should this check `length` too?

File: src/native/components/noticias/NoticiaItem.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <ResponsiveImageView
+        source={{ uri: item.images[0].downloadURL}}
```
Comment (varffs) [inline src/native/components/noticias/NoticiaItem.js]: after all that do we not need the image sizes?

File: src/native/index.js [hunk 4]
```diff
@@ -24,19 +24,16 @@ class Root extends Component {
 
     return(
       <Provider store={store}>
-      {/*
         <PersistGate
           loading={<Loading />}
-        >
-        */}
+          persistor={persistor}
```
Comment (varffs) [inline src/native/index.js:29]: Is this fixing my previous issues?

**Review by varffs — COMMENTED** [review]
All looks fine. Just waiting the questions before merge.

File: src/native/components/noticias/NoticiaItem.js (hunk 2 above)
Comment (mixcas) [reply src/native/components/noticias/NoticiaItem.js]: not sure but will do

File: src/native/index.js (hunk 4 above)
Comment (mixcas) [reply src/native/index.js:29]: I don't think so

File: src/native/components/noticias/NoticiaItem.js (hunk 3 above)
Comment (mikeyrayvon) [reply src/native/components/noticias/NoticiaItem.js]: yea we still need them. but we need to sort out an approach for media query

**Pull request merged**